packages/lesswrong/components/posts/PostsTitle.tsx

```diff
@@ -238,6 +279,16 @@ const PostsTitle = ({
           />
         </InteractionWrapper>
       </span>}
+      {currentUser?.isAdmin && isEAForum && digestTagLink && <InteractionWrapper className={classes.interactionWrapper}>
+          <Link
+            to={digestTagLink}
+            className={classNames(classes.tag, classes.digestTag)}
+            eventProps={{digestNum: digestNum ? `${digestNum}` : ''}}
```

Comment:
It might be better to just change `eventProps` to take a `JsonRecord` instead of `Record<string, string>`